inference/inference.py

```diff
@@ -471,7 +476,8 @@ def autoregressive_inference(params, ic, valid_data_full, model):
     #run autoregressive inference for multiple initial conditions
     for i, ic in enumerate(ics):
       logging.info("Initial condition {} of {}".format(i+1, n_ics))
-      sr, sp, vl, a, au, vc, ac, acu, accland, accsea = autoregressive_inference(params, ic, valid_data_full, model)
+      sr, sp, vl, a, au, vc, ac, acu, accland, accsea, inference_logs = autoregressive_inference(params, ic, valid_data_full, model)
+      del inference_logs
```

Comment:
```suggestion
      sr, sp, vl, a, au, vc, ac, acu, accland, accsea, _ = autoregressive_inference(params, ic, valid_data_full, model)
```
is typical for unused outputs.

Reply:
Yes, I believe you are referring to PEP 640. I picked this up as an alternative way of handling unused variables [here](https://github.com/google/jax-cfd/blob/07e963417c27ce26ba272b31e555d6c21fdc3d5d/jax_cfd/ml/interpolations.py#L61). I like it because it documents the outputs of the function without forcing the user to find the return statement of an untyped function. The `del` is nice to document that the variable is unused (and save on memory).

Reply:
I guess I just don't think that instances of the function call should be responsible for documenting its outputs. Especially if the variable is deleted right away, it's hard to know what its intended use is (I wouldn't put much weight in a placeholder name).

Anyway, doesn't need to hold up this PR—I just find the pattern a little confusing.

In the example you link, these deleted variables should just not be arguments to the init function...